index.html

```diff
@@ -48,7 +57,180 @@ <h1>I’m Nurbol<br>Glad to see<br> you!</h1>
+        <div class="top">
+            <h2 id="six">Tonic</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+        </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="bottom">
+            <ul class="tags">
+            <li>html</li>
+            <li>css</li>
+            <li>javaScript</li></ul>
+            <button class="action">See Project</button>
+        </div>
+    </div>
+</div>
+
+<!--4 work card-->
+
+    <div class="works-card">
+        <img src="worksimg/Snapshoot Portfolio@2x (3).png">
+        <div class="info-block">
+            <div class="top">
+            <h2 id="eight">Multi-Post<br>Stories</h2>
+            <ul class="frame">
+                <li id="client">CANOPY</li>
+                <li id="dot"> </li>
+                <li id="role">Back End Dev</li>
+                <li id="dot"> </li>
+                <li id="year">2015</li></ul>
+            </div>
+            <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <div class="bottom">
+                <ul class="tags">
+                <li>html</li>
+                <li>css</li>
+                <li>javaScript</li></ul>
+                <button class="action">See Project</button>
+            </div>
+        </div>
+    </div>            
+</div>    
+    </section>
+
+<!-- ─── About Me Section ─── -->    
+
+    <section class="aboutme">
+        <div class="text">
+            <h1>About<br>Myself</h1>
+            <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        </div>
+        <div class=links>
+            <button type="button" class="button">LET’S CONNECT</button>
+            <ul class="media-links">
+                <li class="media"><a href="https://twitter.com/everinurmind/"><img src="images\Twitter@2x.png" alt="twitter logo"></a></li>
+                
+                <li class="media"><a href="https://www.linkedin.com/in/everinurmind/"><img src="images\Linkedin@2x.png" alt="linked-in logo"></a></li>
+                
+                <li class="media"><a href="https://medium.com/"><img src="images\Medium@2x.png" alt="medium logo"></a></li>
+                
+                <li class="media" ><a href="https://github.com/everinurmind/"><img src="images\GitHub@2x.png" alt="github logo"></a></li>
+                
+                <li><a class="media" href="https://www.angellist.com/"><img src="images\AngelList@2x.png" alt="angellist logo"></a>
+                </li>
+            </ul>
+            <button class="action2">Get My Resume</button>
```

Comment:
- [ ] How about adding some `box-shadow` when `button` is hovered on by users according to the design too? It makes your design more aesthetic and improves user experience.